src/screens/ExplainSheet.js

```diff
@@ -779,7 +776,7 @@ export const explainers = (params, theme) => {
           {lang.t('explain.obtain_l2_asset.fragment3')}
         </Text>
       ),
-      logo: <ChainBadge chainId={ethereumUtils.getChainIdFromNetwork(params?.network)} marginBottom={8} position="relative" size="large" />,
+      logo: <ChainBadge chainId={fromChainId} marginBottom={8} position="relative" size="large" />,
```

Comment:
just wanted to call out this change: previously was `params?.network` and now is `fromChainId` instead of `chainId`

Reply:
yes this is correct, i changed all of the navigator push events to this ExplainSheet component so it matches correctly